### ISSUE TITLE
Having trouble with 0.4.0 native-osx

### ISSUE THREAD

**Issue by mike-potter** [EugenMayer/docker-sync#347]
Did a "gem install docker-sync" this morning and got the 0.4.0 version.  I did "docker-sync-stack clean" and then changed my volumes from the sync_strategy: "unison" that was working, to use the new "native-osx" to test performance.

I cannot get it to do anything.  Everything starts with no errors, but there isn't any container for the sync volume anymore.  My docker-sync.yml is like this:
```
version: "2"
options:
  verbose: true
syncs:
  d8-osx-sync:
    sync_strategy: 'native_osx'
    src: '.'
    sync_host_ip: 192.168.99.100
    sync_prefer: 'src'
```
and my docker-compose.yml is like this:
```
version: '2.1'
services:
  www:
    image: phase2/apache-php:php70
    volumes:
      - d8-osx-sync:/var/www:nocopy # will be mounted on /var/www
volumes:
  d8-osx-sync:
    external: true
```
In the past when using "unison", a docker-ps would show a running "d8-osx-sync" container.  But with "native_osx" it says it is starting the d8-osx-sync container but "docker ps" never shows it.  And my main www container doesn't show any files in /var/www.

Am I missing something or is there a bug?  I looked at the commits made since 0.4.0 but don't see anything obvious (am already using :nocopy).

Still a bit new at this but willing to help debug.

**Comment by mike-potter** [comment]
Another note: I am using docker-machine if that matters.

**Comment by donnykurnia** [comment]
@mike-potter `native-osx` didn't work with `vboxsf` used by virtualbox. #345 #346

**Comment by EugenMayer** [comment]
@mike-potter please remove:

 sync_host_ip: 192.168.99.100
 sync_prefer: 'src'
sync_strategy: 'native_osx'

they will all default just right for you/

We just releaed 0.4.1 which could be a fix for your case.

In case you are using docker-machine with virtualbox, please see https://github.com/EugenMayer/docker-sync/issues/346

**Comment by mike-potter** [comment]
Yes, we are using virtualbox and cannot change that unfortunately.  Thanks for the quick response but will stick with unison then.  Sad this was a virtualbox issue.

(and yes, the new 'auto' for docker-machines different from 'default' is working great, so thanks for that too)

**Comment by EugenMayer** [comment]
closed due to  #346 - further notice, fusion would not work either
